src/amo/components/UserProfileEdit/index.js

```diff
@@ -143,13 +143,23 @@ export class UserProfileEditBase extends React.Component<Props, State> {
         pictureData: null,
         successMessage: i18n.gettext('Profile successfully updated'),
       });
+      window.scroll({
```

Comment:
We need to use a `prop` and not `window` directly, for example by doing this: https://github.com/mozilla/addons-frontend/blob/24fd4a97a6512cdd2f18ec534400a3360682fdee/src/amo/components/AddAddonToCollection/index.js#L58-L60

Reply:
Then you can use `this.props._window`.